index.js

```diff
@@ -52,4 +52,117 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+// Element selectors
+
+const shopItemList = document.querySelector(".store--item-list");
+const cartItemList = document.querySelector(".cart--item-list");
+const totalPrice = document.querySelector('.total-number');
+
+// Program Logic
+
+function addItemToCart(item) {
+  if (!state.cart.includes(item)) {
+    item.quantity = 1;
+    state.cart.push(item);
+    UpdateTotal();
+    // console.log(item.quantity)
+  }  else {
+    item.quantity++;
+    UpdateTotal();
+  }
+  // console.log(item.quantity)
+  renderCart(); 
+}
+
+// Rendering
+
+function renderShop () {
+  state.items.forEach((item) => {
+    const li = document.createElement('li');
+    
+    const div = document.createElement('div');
+    div.setAttribute("class", "store--item-icon");
+
+    const img = document.createElement('img');
+    img.src = 'assets/icons/' + item.id + '.svg';
+    img.alt = item.name;
+    
+    const addToCart = document.createElement('button');
+    addToCart.innerHTML = 'Add to cart';
+    addToCart.addEventListener('click', () => addItemToCart(item));
+
+    div.append(img);
+    li.append(div);
+    li.append(addToCart);
+
+    shopItemList.append(li);
+
+
+  })
+}
+
+function renderCart () {
+
+  cartItemList.innerHTML = '';
+
+  state.cart.forEach((item) => {
+    if (item.quantity > 0) {
+      const li = document.createElement('li');
+
+      const img = document.createElement('img');
+      img.setAttribute('class', 'cart--item-icon');
+      img.src = 'assets/icons/' + item.id + '.svg';
+      img.alt = item.name;
+
+      const text = document.createElement('p');
+      text.innerHTML = item.name;
+      
+      const removeButt = document.createElement('button');
+      removeButt.setAttribute('class', 'remove-btn');
+      removeButt.innerHTML = '-';
+      removeButt.addEventListener('click', () => {
+        item.quantity--;
+        renderCart();
+        UpdateTotal();
+      })
+
+      const span = document.createElement('span');
+      span.innerHTML = item.quantity;
+
+      const addButt = document.createElement('button');
+      addButt.setAttribute('class', 'add-btn');
+      addButt.innerHTML = '+';
+      addButt.addEventListener('click', () => {
+        item.quantity++;
+        renderCart();
+        UpdateTotal();
+      })
+
+      li.append(img);
```

Comment:
appending everything together at the end makes it much more readable, i believe you could also append things on the same line for example: li.append(img, text, removeButt, span, addButt);